### PR TITLE
Fix #180: Fix production errors

### LINKED ISSUE #180: Errors on production
```
TypeError: Cannot read property 'endsWith' of undefined
  at /app/lib/periodicChecks.js:157:31
  at Array.some (<anonymous>)
  at /app/lib/periodicChecks.js:156:51
  at Array.filter (<anonymous>)
  at Object.ensureAllIssuesHaveProjects (/app/lib/periodicChecks.js:155:47)
  at async Promise.all (index 1)
  at async runChecks (/app/index.js:129:13)
  at async /app/index.js:181:7
```

```
 09:53:42.384Z ERROR event: Cannot read property 'split' of undefined (id=6374b3f0-4ce0-11eb-92bf-82d148a0655f)
Jan 02 01:53:42   TypeError: Cannot read property 'split' of undefined
Jan 02 01:53:42       at getNewItemsFromFileByRegex (/app/lib/utils.js:79:35)
Jan 02 01:53:42       at getNewJobsFromFile (/app/lib/checkPullRequestJob.js:46:10)
Jan 02 01:53:42       at addsNewJob (/app/lib/checkPullRequestJob.js:53:19)
Jan 02 01:53:42       at /app/lib/checkPullRequestJob.js:189:45
Jan 02 01:53:42       at Array.filter (<anonymous>)
Jan 02 01:53:42       at Object.checkForNewJob (/app/lib/checkPullRequestJob.js:188:38)
Jan 02 01:53:42       at async runChecks (/app/index.js:91:13)
Jan 02 01:53:42       at async /app/index.js:233:7
Jan 02 01:53:42   --
Jan 02 01:53:42   event: {
Jan 02 01:53:42     "event": "pull_request.synchronize",
Jan 02 01:53:42     "id": "6374b3f0-4ce0-11eb-92bf-82d148a0655f",
Jan 02 01:53:42     "installation": 213174,
Jan 02 01:53:42     "repository": "oppia/oppia"
Jan 02 01:53:42   }
```

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Fixes #180: Fixes errors on production which were listed in the issue. 

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/periodicChecks.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ const ensureAllIssuesHaveProjects = async (context) => {
 
   const issuesWithoutProjects = allOpenIssues.filter((issue) => {
     const issueBelongsToProject = allProjectCards.some((card) => {
-      return card.content_url.endsWith(issue.number);
+      return card.content_url && card.content_url.endsWith(issue.number);
```
Comment (vojtechjelinek) [inline lib/periodicChecks.js:157]: Why can `content_url` be undefined?

File: lib/utils.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ const getNewItemsFromFileByRegex = (regex, file) => {
   // This is done because classes can only be created in a new line.
   // '+' represents an addition while '-' represents a deletion.
   const newLine = '\n';
-  const changesArray = file.patch.split(newLine);
+  const changesArray = file.patch ? file.patch.split(newLine) : [];
```
Comment (vojtechjelinek) [inline lib/utils.js:83]: Why can `patch` be undefined?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! I have two comments.

File: lib/periodicChecks.js (hunk 1 above)
Comment (jameesjohn) [reply lib/periodicChecks.js:157]: When a project card hasn't been linked to an issue. For example, if the project card just contains notes.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**